### PR TITLE
Fix `exclude-path` arguments on link checker action

### PR DESCRIPTION
**Pull request by RuiL1904**

### CODE REVIEW

**Review by MarioRodrigues10 — APPROVED** [review]

**Comment by CarlosHSF99** [comment]
I can't see why https://twitter.com/ is timing out, but if it is then we should probably add it to the `.lycheeignore` to prevent it from becoming a problem again in the future.

Does it make sense to exclude the whole path or is it sufficient to exclude the link?

**Comment by nelsonmestevao** [comment]
Great idea @CarlosHSF99 🙌

**Comment by CarlosHSF99** [comment]
Fixed .lycheeignore to use regex and added https://twitter.com (`https:\/\/twitter\.com`) to it.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,14 +37,15 @@ jobs:
...
+    timeout-minutes: 2
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
       - name: Link Checker
         uses: lycheeverse/lychee-action@v1
         with:
-          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx packages/ui/components/landing/Footer/config.tsx './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
+          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx --exclude-path packages/ui/components/landing/Footer/config.tsx './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
```
Comment (nelsonmestevao) [inline .github/workflows/ci.yml:48]:
```suggestion
          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx  './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
```

Should we remove Footer config from exclude file then? @CarlosHSF99 @RuiL1904

Comment (CarlosHSF99) [reply .github/workflows/ci.yml:48]: Although all the links in that file are valid, their purpose is to be build a larger link so I think it does make sense to exclude it.

**Review by nelsonmestevao — APPROVED** [review]

**Review by CarlosHSF99 — APPROVED** [review]

**Pull request merged**